File: RedSheet.DbEntities/Models/vLiteRelationshipRequire.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteRelationshipRequires", Schema = "dbo")]
	[Serializable]
	public partial class vLiteRelationshipRequire  
	{
 
	[Required] 
	[MaxLength(200)]
	public string ClassName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LiteRelationshipRequireId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string LiteRelationshipRequireName { get; set; } 

	public vLiteRelationshipRequire()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-lite-relationship-require.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteRelationshipRequire {
    constructor(vLiteRelationshipRequire?: vLiteRelationshipRequire )  {
        let properties = [ "className", "liteRelationshipRequireId", "liteRelationshipRequireName",];
        for (let property of properties)
            if (vLiteRelationshipRequire && vLiteRelationshipRequire[property])
                this[property] = vLiteRelationshipRequire[property];
    }
 
    @required()
    @maxLength(200)
	className : string =   undefined;
 
	liteRelationshipRequireId : number =   0 ;
 
    @required()
    @maxLength(200)
	liteRelationshipRequireName : string =   undefined;


}
